provider/cmd/crd2pulumi/gen/python.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gen
+
+import (
+	"bytes"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/pulumi/pulumi/pkg/v2/codegen/python"
+	pschema "github.com/pulumi/pulumi/pkg/v2/codegen/schema"
+	"github.com/pulumi/pulumi/sdk/v2/go/common/util/contract"
+)
+
+func (pg *PackageGenerator) genPython(types map[string]pschema.ObjectTypeSpec, baseRefs []string) (map[string]*bytes.Buffer, error) {
+	pkg, err := getPackage(types, baseRefs)
+	if err != nil {
+		return nil, errors.Wrapf(err, "generating package")
+	}
+	pkg.Language["python"] = rawMessage(map[string]interface{}{
+		"compatibility":       "kubernetes20",
+		"moduleNameOverrides": pg.moduleToPackage(),
+		"requires": map[string]string{
+			"pulumi":   "\u003e=2.0.0,\u003c3.0.0",
+			"pyyaml":   "\u003e=5.1,\u003c5.2",
+			"requests": "\u003e=2.21.0,\u003c2.22.0",
+		},
+	})
+
+	files, err := python.GeneratePackage(tool, pkg, nil)
```

Comment:
We can skip emitting `provider.py` and the empty `README.md`.

I think we're going to want to be able to customize the generated `setup.py`. First, it has an `InstallPluginCommand` that isn't relevant here that we need to strip.

Second, it has `${VERSION}` which is intended to be replaced as part of the build process. For https://github.com/pulumi/pulumi-kubernetes/issues/1261, we'll prob need it for each package we generate, but for people just running the tool on their own, it's probably not needed. For now, it's probably fine to leave it as-is, we'll just need to decide what we want to do here. We can chat about it in our 1:1.